---
description: "자세한 정보: ' <labelname> <labelname> '이 (가)이 문을 포함 하지 않는 ' Try ', ' Catch ' 또는 ' Finally ' 문 내부에 있으므로 ' GoTo '는 사용할 수 없습니다."
title: "'<labelname>'이 이 문을 포함하지 않는 'Try', 'Catch' 또는 'Finally' 문 내부에 있으므로 'GoTo <labelname>'은 사용할 수 없습니다."
ms.date: 07/20/2015
f1_keywords:
- bc30754
- vbc30754
helpviewer_keywords:
- BC30754
ms.assetid: 2eefc7fb-fdf0-41e9-bf60-c3bc93580e14
ms.openlocfilehash: 952612fbac2304f57cf2d0a7ac6e701f1d5bb054
ms.sourcegitcommit: 10e719780594efc781b15295e499c66f316068b8
ms.translationtype: MT
ms.contentlocale: ko-KR
ms.lasthandoff: 02/14/2021
ms.locfileid: "100437412"
---
# <a name="goto-labelname-is-not-valid-because-labelname-is-inside-a-try-catch-or-finally-statement-that-does-not-contain-this-statement"></a>'\<labelname>'이 이 문을 포함하지 않는 'Try', 'Catch' 또는 'Finally' 문 내부에 있으므로 'GoTo \<labelname>'은 사용할 수 없습니다.

`Try...Catch...Finally` 블록으로 분기할 수 없습니다.  
  
 **오류 ID:** BC30754  
  
## <a name="to-correct-this-error"></a>이 오류를 해결하려면  
  
- 레이블이 `Try...Catch...Finally` 블록 앞에 오도록 코드 구조를 바꿉니다.  
  
## <a name="see-also"></a>참고 항목

- [Try...Catch...Finally 문](../language-reference/statements/try-catch-finally-statement.md)
